ckan/tests/config/test_middleware.py

```diff
@@ -246,12 +236,14 @@ def test_ask_around_pylons_extension_route_get_before_map(self):
         }
         wsgiref.util.setup_testing_defaults(environ)
 
-        answers = app.ask_around('can_handle_request', environ)
+        answers = app.ask_around(environ)
 
-        eq_(len(answers), 1)
-        eq_(answers[0][0], True)
-        eq_(answers[0][1], 'pylons_app')
-        eq_(answers[0][2], 'extension')
+        eq_(len(answers), 2)
+        eq_(answers[0][0], False)
+        eq_(answers[0][1], 'flask_app')
+        eq_(answers[1][0], True)
+        eq_(answers[1][1], 'pylons_app')
+        eq_(answers[1][2], 'extension')
```

Comment:
why not just `eq_(answers, [(False, 'flask_app'), (True, 'pylons_app', 'extension')])`?  I think you get pretty-printed diffs of the lists when they don't match that way (and it's a shorter test)